packages/babel-parser/src/parser/statement.ts

```diff
@@ -349,6 +348,19 @@ export default abstract class StatementParser extends ExpressionParser {
     }
   }
 
+  startsAwaitUsing(): boolean {
+    let next = this.nextTokenInLineStart();
+    if (this.isUnparsedContextual(next, "using")) {
+      next = this.nextTokenInLineStartSince(next + 5);
+      const nextCh = this.codePointAtPos(next);
+      if (this.chStartsBindingIdentifier(nextCh, next)) {
```

Comment:
Test case (to test both with the plugin enabled and disabled):
```js
await using instanceof x
```

Reply:
Good catch, added new test cases.